.github/workflows/test-ignore-files.yml

```diff
@@ -0,0 +1,13 @@
+name: Test
```

Comment:
We can probably merge both workflows and have them in one place to make things shorter/cleaner.

Reply:
If we merge them if either one of them fails, we won't know until we dig deeper which scenario failed. I prefer to keep them separate so I can just glance and know which one failed, if that makes sense.

Reply:
With multiple jobs, we can see which one passed or not in the same workflow. It'll be longer but can be easier to check, IMO.

Reply:
Consider this: if we have two of these (let's call A and B) then when we have them in the same file, there must be an order which must be followed. Say first A then B - in this scenario, if A fails we don't know whether B is working as expected because the whole workflow stops execution. That's why I'm against it, and unless you're worried about the runner costs here, I think it should be okay to keep them separate... What do you think? I can place them in a single file easily just wanted to ask your opinion.